internal/docker/docker.go

```diff
@@ -337,6 +346,96 @@ func (c *Docker) runCommand(cmdName string, args ...string) error {
 	return err
 }
 
+// setupPartialRepo copies the following files from the [config.Config.Repo] to
+// partialRepoDir in the given ReleaseRequest:
+//
+// 1. all directories that make up all libraries, or one library, if the library
+// ID is specified.
+//
+// 2. the .librarian directory.
+//
+// 3. global files declared in config.yaml.
+func setupPartialRepo(request *ReleaseRequest) error {
+	if request.partialRepoDir == "" {
+		request.partialRepoDir = filepath.Join(os.TempDir())
```

Comment:
can we use `filepath.Join(ctg.WorkRoot, "release-init")`

Reply:
Done.